packages/spatial/src/physics/classes/Physics.test.ts

```diff
@@ -361,3 +362,714 @@ describe('Physics', () => {
+
+    beforeEach(async () => {
+      createEngine()
+      await Physics.load()
+      physicsWorld  = Physics.createWorld()
+      getMutableState(PhysicsState).physicsWorld!.set(physicsWorld!)
+      physicsWorld!.timestep = 1 / 60
+
+      // Create the entity
+      testEntity = createEntity()
+      setComponent(testEntity, TransformComponent)
+      setComponent(testEntity, RigidBodyComponent, { type: BodyTypes.Dynamic })
+      Physics.createRigidBody(testEntity, physicsWorld!)
+    })
+
+    afterEach(() => {
+      removeEntity(testEntity)
+      physicsWorld = undefined
+      return destroyEngine()
+    })
+
+    it("should apply the impulse to the RigidBody of the entity", () => {
+      const testImpulse = new Vector3(1,2,3)
+      const body = Physics._Rigidbodies.get(testEntity)!
+      Physics.applyImpulse(testEntity, testImpulse)
+    })
+  })
+  */
+
+    /**
+  // @todo How to check rotations?
+  describe("lockRotations", () => {
+    let testEntity = UndefinedEntity
+    let physicsWorld :World | undefined= undefined
+
+    beforeEach(async () => {
+      createEngine()
+      await Physics.load()
+      physicsWorld  = Physics.createWorld()
+      getMutableState(PhysicsState).physicsWorld!.set(physicsWorld!)
+      physicsWorld!.timestep = 1 / 60
+
+      // Create the entity
+      testEntity = createEntity()
+      setComponent(testEntity, TransformComponent)
+      setComponent(testEntity, RigidBodyComponent, { type: BodyTypes.Dynamic })
+      Physics.createRigidBody(testEntity, physicsWorld!)
+    })
+
+    afterEach(() => {
+      removeEntity(testEntity)
+      physicsWorld = undefined
+      return destroyEngine()
+    })
+
+    it("should lock rotations on the entity", () => {
+      const ExpectedValue = {x: 4, y: 3, z: 2, w: 1} as Rotation
+      const body = Physics._Rigidbodies.get(testEntity)!
+      assert.notDeepEqual(body.rotation(), ExpectedValue)
+      Physics.lockRotations(testEntity, true)
+      body.setRotation(ExpectedValue, false)
+      assert.notDeepEqual(body.rotation(), ExpectedValue)
+    })
+
+    it("should disable locked rotations on the entity", () => {
+      const ExpectedValue = {x: 4, y: 3, z: 2, w: 1} as Rotation
+      const body = Physics._Rigidbodies.get(testEntity)!
+      assert.notDeepEqual(body.rotation(), ExpectedValue)
+      Physics.lockRotations(testEntity, true)
+      body.setRotation(ExpectedValue, false)
+      assert.notDeepEqual(body.rotation(), ExpectedValue)
+      Physics.lockRotations(testEntity, true)
+      assert.deepEqual(body.rotation(), ExpectedValue)
+    })
+  })
+  */
+
+    /**
+  // @todo How to check rotations?
+  describe("setEnabledRotations", () => {})
+  describe("updatePreviousRigidbodyPose", () => {})
+  describe("updateRigidbodyPose", () => {})
+  describe("setKinematicRigidbodyPose", () => {})
+  */
+  }) // << Rigidbodies
+
+  describe('Colliders', () => {
+    /**
+    // @todo How to check for `CollisionGroups` behavior?
```

Comment:
you can use the collisionGroups() and isSensor() functions on the collider object

Reply:
ty. Solved in 9e63efe3bee15f048d18d911eec260ef5f12eccb